src/locking/LockingManager.ts

```diff
@@ -17,34 +16,15 @@ const MODULE_NAME = 'LockingManager';
  *  - E = mutually exclusive
  */
 export default class LockingManager {
-  public static createExclusiveLock(tenantID: string, entity: LockEntity, key: string, expirationTimeSeconds?: number): Lock {
-    return this.createLock(tenantID, entity, key, LockType.EXCLUSIVE, expirationTimeSeconds);
+  public static createExclusiveLock(tenantID: string, entity: LockEntity, key: string, lockValiditySecs?: number): Lock {
+    return this.createLock(tenantID, entity, key, LockType.EXCLUSIVE, lockValiditySecs);
   }
 
-  public static async acquire(lock: Lock, timeout?: number): Promise<boolean> {
+  public static async acquire(lock: Lock, timeoutMs?: number, retry = true): Promise<boolean> {
```

Comment:
Putting timeoutMs default to `= 0` will make the API cleaner and the code less complex to understand.

Reply:
Ok